src/ast/passes/codegen_llvm.cpp

```diff
@@ -1336,6 +1336,38 @@ void CodegenLLVM::binop_string(Binop &binop)
                            inverse);
 }
 
+void CodegenLLVM::binop_integer_array(Binop &binop)
+{
+  if (binop.op != Operator::EQ && binop.op != Operator::NE)
+  {
+    LOG(FATAL) << "missing codegen to array operator \"" << opstr(binop)
+               << "\"";
+  }
+
+  std::string string_literal;
+
+  // cmp returns 0 when arrays are equal
+  bool inverse = binop.op == Operator::EQ;
+
+  auto left_array_ptr = getString(binop.left);
+  auto right_array_ptr = getString(binop.right);
```

Comment:
This doesn't seem correct, if you want to work with integer arrays the why use `getString`?